### ISSUE TITLE
Crashed blender

### ISSUE THREAD

**Issue by kaushikbm** [node-dojo/dojo-recursive-bins#10]
I'm using Blender 3.5. It crashes while opening the file. Is this a bug or some other issue ?

**Comment by BGerwe** [comment]
Same here. I'm on Blender 3.5.1, Windows 11 23H2.

I start up Blender then go to open either `Dojo Bin Generator_recursive 2 step_v0.1.1` or `Dojo Bin Generator_recursive red bins_v.0.1.1` and it just crashes.

I repeated this after starting Blender with debug options via `blender_debug_log.cmd` and got the following log:
> I0508 13:21:21.501030 28604 device.cpp:47] CUEW initialization failed: Error opening the library
I0508 13:21:21.516103 28604 device.cpp:32] HIPEW initialization succeeded
I0508 13:21:21.516103 28604 device.cpp:34] Found precompiled kernels
WARN (bke.customdata): C:\Users\blender\git\blender-v350\blender.git\source\blender\blenkernel\intern\customdata.cc:4319 CustomData_layer_ensure_data_exists: CustomDataLayer->data is NULL for type 46.
WARN (bke.customdata): C:\Users\blender\git\blender-v350\blender.git\source\blender\blenkernel\intern\customdata.cc:4319 CustomData_layer_ensure_data_exists: CustomDataLayer->data is NULL for type 46.
Error   : EXCEPTION_ACCESS_VIOLATION
Address : 0x00007FF7B37DE1F2
Module  : blender.exe
Thread  : 00006fbc
Switching to fully guarded memory allocator.
Blender 3.5.1
Build: 2023-04-24 23:56:35 Windows release
argv[0] = C:\Program Files\Blender Foundation\Blender 3.5\\blender
argv[1] = --debug
argv[2] = --debug-cycles
argv[3] = --python-expr
argv[4] = import bpy; bpy.ops.wm.sysinfo(filepath=r'C:\Users\<user>\AppData\Local\Temp\blender\debug_logs\blender_system_info.txt')
Read prefs: C:\Users\<user>\AppData\Roaming\Blender Foundation\Blender\3.5\config\userpref.blend
Read blend: C:\Users\<user>\Downloads\Dojo Bin Generator_recursive red bins_v.0.1.1.blend
Writing: C:\Users\<user>\AppData\Local\Temp\blender.crash.txt

Also here's my [blender_system_info.txt](https://github.com/node-dojo/dojo-recursive-bins/files/15254447/blender_system_info.txt)

I looked up some phrases from the log above and found [this post](https://projects.blender.org/blender/blender/issues/96870), which seems like the problem arises with non-NVIDIA GPUs. Personally, I'm on an AMD Radeon 6650XT. @kaushikbm, are you also using a non-NVIDIA GPU?

Following suggestions from that post I just tried updating my GPU driver, and also uninstalled the driver then restarted my system and neither worked.

**Comment by BGerwe** [comment]
Also tried on a different system with an NVIDIA card—still crashed. 
[blender_debug_output.txt](https://github.com/node-dojo/dojo-recursive-bins/files/15254751/blender_debug_output.txt)

**Comment by doneskies** [comment]
You may have to use a more updated version of Blender. I just opened it in 4.1 without any issue

**Comment by BGerwe** [comment]
> You may have to use a more updated version of Blender. I just opened it in 4.1 without any issue

Yup, I think that's the answer. I just installed v3.6.8 and was able to open the files.
